CONTRIBUTING.md

```diff
@@ -110,31 +110,41 @@ All contributions to the What The Hack repo come through pull requests. This mea
 
 Okay, ready to get started creating your own What The Hack?
 
-The instructions below assume you have the [Git command line tool](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git) on your machine. If you're more comfortable in a GUI Git client, you can use that too (we recommend [Atlassian's SourceTree](https://www.sourcetreeapp.com/)).
+We have a GitHub Action that automates the creation of a new hack and "scaffolds" out all of the Markdown templates from the [WTH Author's Guide](https://aka.ms/wthauthor) for you. After that, you can open your [favorite text editor](https://code.visualstudio.com) and start plugging in your content.
+
+The instructions below assume you have the [Git command line tool](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git) and [Visual Studio Code](https://code.visualstudio.com/) installed on your machine. 
 
 1. Create a fork of the WTH repo
-   - Navigate to the WTH repo at: <https://aka.ms/wthrepo>
+   - Navigate to the WTH repo at: [https://aka.ms/wthrepo](https://aka.ms/wthrepo)
    - Click the "Fork" button at the top right of the page and then choose the account you want to create the fork in. 
-1. Clone your new fork to your local machine
+1. Navigate to your fork of the WTH repo in the browser at: 
+    ```https://github.com/<myGithubName>/WhatTheHack```
+1. Click on the "Actions" tab at the top of the page and then select "Create new hack" from the list of "All Workflows" that appears on the left side of the screen.
```

Comment:
When you fork a repo, the workflows will initially be disabled for safety reasons.  Below is what they will see:

"Workflows aren’t being run on this forked repository
Because this repository contained workflow files when it was forked, we have disabled them from running on this fork. Make sure you understand the configured workflows and their expected usage before enabling Actions on this repository."

They will see this when they click "Actions" tab.  They will need to also click "I understand my workflows, go ahead and enable them" before proceeding to select "Create new hack" from the list.

Reply:
Good catch!  I observed this today when I had the SAP authors try to run it.  It's probably better if I get a screenshot of this message.